editor/register_editor_types.cpp

```diff
@@ -272,6 +272,8 @@ void register_editor_types() {
 	GLOBAL_DEF("editor/import/reimport_missing_imported_files", true);
 	GLOBAL_DEF("editor/import/use_multiple_threads", true);
 
+	GLOBAL_DEF(PropertyInfo(Variant::INT, "editor/import/atlas_max_width", PROPERTY_HINT_RANGE, "128,8192,1,or_greater"), 2048);
```

Comment:
I set the minimum size to `128`, I think that makes sense as a lower bound, we can go with the `1` value but it's not gonna be very useful